draft-ietf-ice-rfc5245bis.xml

```diff
@@ -534,16 +533,16 @@ better than one with higher priority.
 <t>
 Consequently, ICE assigns one of the agents in the role of the
 CONTROLLING AGENT, and the other of the CONTROLLED AGENT. The
-controlling agent gets to nominate which candidate pairs will get used
-for media amongst the ones that are valid.
+controlling agent nominates a candidate pair from the valid
+candidate pairs to be used for media.
 </t>
 
 <t>
 When nominating, the controlling agent lets the checks continue
 until at least one valid candidate pair for each media stream is
-found. Then, it picks amongst those that are valid, and sends a second
-STUN request on its NOMINATED candidate pair, but this time with a
-flag set to tell the peer that this pair has been nominated for use.
+found and then selects a candidate pair from the valid candidate pairs
+and sends a STUN request on the selected pair with a flag set to indicate
+to the controlled peer that it has nominated the selected pair.
 This is shown in <xref target="fig-regular-select"/>.
 </t>
 
```

Comment:
I am not sure using "select" terminology is good here, because it confuses with the "selected pair" concept. I would like to "picks", or use some other terminology (e.g., "chooses").